code/game/gamemodes/roleset/faction/excelsior.dm

```diff
@@ -13,4 +13,10 @@
 	req_crew = 6
 	leaders = -1 //Every excelsior spawned directly is a leader. Non leaders are those recruited during gameplay
 
+	story_ineligible = list(JOBS_SECURITY, JOBS_COMMAND)
 
+// Code to prevent a role from being picked by the storyteller.
+/datum/storyevent/roleset/faction/excelsior/antagonist_suitable(var/datum/mind/player, var/datum/antagonist/antag)
```

Comment:
Move this code up to `/datum/storyevent/roleset/faction`, so `story_ineligible` actually works for other antags.

Reply:
Done, not sure about the story_ineligible being unique for each factions.